test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Actions/UIActionEngineTests.cs

```diff
@@ -140,6 +140,81 @@ public async Task GetPreviewResultsAsync_WithMultipleActions_SortsPackageIdentit
             Assert.Equal(packageIdentityC.Id, addedResults[2].Id);
         }
 
+        [Theory]
+        [MemberData(nameof(GetInstallActionTestData))]
+        public async Task CreateInstallAction_OnInstallingProject_WithNewSourceMapping_DoesNotLogTelemetry(ContractsItemFilter activeTab, bool isSolutionLevel, string packageIdToInstall, bool? expectedPkgWasTransitive)
+        {
+            // Arrange
+            var telemetrySession = new Mock<ITelemetrySession>();
+            TelemetryEvent lastTelemetryEvent = null;
+            telemetrySession
+                .Setup(x => x.PostEvent(It.IsAny<TelemetryEvent>()))
+                .Callback<TelemetryEvent>(x => lastTelemetryEvent = x);
+            var telemetryService = new NuGetVSTelemetryService(telemetrySession.Object);
+            TelemetryActivity.NuGetTelemetryService = telemetryService;
+
+            var sourceProvider = new Mock<ISourceRepositoryProvider>();
+            var settings = new Mock<ISettings>();
+            var nugetPM = new NuGetPackageManager(sourceProvider.Object, settings.Object, @"\packagesFolder");
```

Comment:
I tried the refactoring and got this error.
` System.ArgumentException : 'rootDirectory' must contain an absolute path '.\packagesFolder'.`

Since this test pattern was copied from an existing test in this class, I've filed this issue to find a way to refactor both of them in a separate PR: https://github.com/NuGet/Home/issues/12556